tileParser.g4

```diff
@@ -19,18 +19,15 @@ statements
     ;
 
 statement
-    : loopStmt
-    | blockStmt
+    : expressionStmt
+    | loopStmt
     | selectionStmt
-    | expressionStmt
+    | funcDefStmt
+    | blockStmt
     ; 
     // TODO: create these statements
-    // : expression_stmt
     // | func_def_stmt
     // | return_stmt
-    // | loop_stmt
-    // | selection_stmt
-    // | block_stmt
     // ;
 
 expressionStmt
```

Comment:
could you complete expressionStmt as expression ';' and commit that to this pull request as well, at least it generates the files.